tests/subsys/canbus/isotp/conformance/src/main.c

```diff
@@ -323,7 +323,7 @@ static void prepare_cf_frames(struct frame_desired *frames, size_t frames_cnt,
 	}
 }
 
-static void test_send_sf(void)
+ZTEST(isotp_conformance_tests, test_send_sf)
```

Comment:
```suggestion
ZTEST(isotp_conformance, test_send_sf)
```

All over. With your current PR, the tests in this suite wont even be executed.

Reply:
Sorry. I will modify and verify it again.